CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.1
```

Comment:
version should be 3.1 since its not a bug fix

Reply:
fixed